### PR TITLE
OpenVINO model optimizer component and example pipeline

### PR DESCRIPTION
**Pull request by dtrawins**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/178)

### CODE REVIEW

**Comment by vicaire** [comment]
Hi @dtrawins, 

The comments from one of the robots above provide details instructions on how to pass the CLA check. 

See the comment starting with: 

"Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA)."

Thanks!

**Comment by dtrawins** [comment]
I should be now in the required CLA group. Could you revalidate it now?

File: components/openvino/model_convert/README.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
+# Model optimization component
```
Comment (vicaire) [inline components/openvino/model_convert/README.md]: Would it be possible to move the code to /contrib/components/openvino?

Comment (dtrawins) [reply components/openvino/model_convert/README.md]: sure, moved according to this suggestion

File: samples/openvino/model_optimizer/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# OpenVINO model optimizer pipeline
+
+This is an example of a one step pipeline implementation of model optimization using OpenVINO toolkit
+
+It performs graph optimization and generates Intermediate Representation model format which can be used 
+later by the Inference Engine.
```
Comment (vicaire) [inline samples/openvino/model_optimizer/README.md]: This file is still in the /samples directory. Could we move it to the /contrib directory?

File: samples/openvino/model_optimizer/convert_model_pipeline.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
+import kfp.dsl as dsl
```
Comment (vicaire) [inline samples/openvino/model_optimizer/convert_model_pipeline.py]: This file is still in the /samples directory. Could we move it to the /contrib directory?

Comment (dtrawins) [reply samples/openvino/model_optimizer/convert_model_pipeline.py]: samples are now also moved to contrib

File: samples/openvino/model_optimizer/README.md (hunk 2 above)
Comment (dtrawins) [reply samples/openvino/model_optimizer/README.md]: samples are now also moved to contrib

**Comment by vicaire** [comment]
Bypassing the e2e tests since they are not affected by these changes.

**Pull request merged**